spec/system/reports/admin_sees_report_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe 'Admin visualiza denúncia' do
+  it 'de um post com sucesso' do
+    admin = create(:user, :admin)
+    post = create(:post, :published)
+    create(:report, :for_post)
+    report = create(:report, reportable: post)
+    create(:report, :for_comment)
+
+    login_as admin
+    visit reports_path
+    page.all('.see_more').to_a.second.click
+
+    expect(page).to have_current_path report_path(report)
+    expect(page).to have_content report.reportable.title
+    expect(page).to have_content report.message
+    expect(page).to have_content "Criado por #{report.reportable.user.full_name}"
```

Comment:
Pessoal, agora que eu percebi, nós estamos seguindo as práticas de teste que o João recomendou, de deixar o teste auto contido, ou seja, se estamos esperando uma mensagem de estar na tela, devemos declarar explicitamente no teste qual é essa mensagem. Os dados default do factory só seriam utilizados caso precisássemos criar uma instância de algo sem precisar fazer uma asserção sobre o conteúdo.

Todo nosso projeto está assim, e aí acaba quebrando o padrão nosso.

De mais, está muito bom o trabalho de vcs :)
```suggestion
    admin = create(:user, :admin)
    post = create(:post, :published)
    create(:report, :for_post)
    report = create(:report, reportable: post, message: 'Conteúdo pornográfico')
    create(:report, :for_comment)

    login_as admin
    visit reports_path
    page.all('.see_more').to_a.second.click

    expect(page).to have_current_path report_path(report)
    expect(page).to have_content report.reportable.title
    expect(page).to have_content 'Conteúdo pornográfico'
    expect(page).to have_content "Criado por #{report.reportable.user.full_name}"
```